CONTRIBUTING.md

```diff
@@ -18,20 +18,24 @@ If you're new to open-source development, we recommend going through the GitHub
 We follow something called a "fork and pull request" workflow for collaborating on our project. See [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) for a great overview on what some of these mysterious terms mean! 
 
 Once you've 'forked' and 'cloned' the code to your local machine, please follow these steps:
+
+```shell
+# change location to the root directory of the superduperdb repository
+cd /path/to/superduperdb/root
+
+# create your Python virtual environment
+python3 -m venv .venv
+
+# activate the Python virtual environment
+. .venv/bin/activate  
```

Comment:
It can help users run it directly in Pycharm when we add lang in code block
<img width="945" alt="image" src="https://github.com/SuperDuperDB/superduperdb/assets/15820530/c09959f5-7227-43bb-ab2b-c5e4f70a2560">